javatests/arcs/core/allocator/AllocatorTest.kt

```diff
@@ -96,25 +95,25 @@ class AllocatorTest {
             hostRegistry = ExplicitHostRegistry()
             hostRegistry.registerHost(ReadingHost())
             hostRegistry.registerHost(WritingHost())
-            writePersonHandleConnectionSpec =
-                HandleConnectionSpec(null, personEntityType)
+            writePersonHandleConnection =
+                HandleConnection(null, personEntityType)
 
-            writePersonParticleSpec = ParticleSpec(
-                "WritePerson",
-                WritePerson::class.java.getCanonicalName()!!,
-                mapOf("recipePerson" to writePersonHandleConnectionSpec)
+            writePersonParticle = arcs.core.data.Plan.Particle(
+                    "WritePerson",
+                    WritePerson::class.java.getCanonicalName()!!,
+                    mapOf("recipePerson" to writePersonHandleConnection)
```

Comment:
Indenting is wrong. (You should fix your IDE: Editor -> Code style -> Kotlin -> Tabs and Indents -> set everything to 4

Reply:
Thanks, done.